proposals/csharp-9.0/native-integers.md

```diff
@@ -45,12 +45,25 @@ The tables below cover the conversions between special types.
 
 General notes:
 - Most docs explain `conv.u` as conversion to __native unsigned integer__ and `conv.i` as conversion to __native integer__. But a more appropriate description is that `conv.u` is zero-extending and `conv.i` is sign-extending (if the most significant bit is `1`, then the conversion fills with `1`s).
-- Some non-obvious choices are guided by pre-existing semantics. For example, `sbyte->nint` uses `conv.i` and `byte->nint` uses `conv.u`, the same way that `sbyte->long` uses `conv.i8` and `byte->long` uses `conv.u8`. Also, `checked(void*->nint)` uses `conv.ovf.i.un` the same way that `checked(void*->long)` uses `conv.ovf.i8.un`.
+- `checked` contexts for both **widening** and **narrowing** are:
+  - `conv.ovf.*` for `signed to *`
+  - `conv.ovf.*.un` for `unsigned to *`
+- `unchecked` contexts for **widening** are:
+  - `conv.i*` for `signed to *` (where * is the target width)
+  - `conv.u*` for `unsigned to *` (where * is the target width)
+- `unchecked` contexts for **narrowing** are:
+  - `conv.i*` for `any to signed *` (where * is the target width)
+  - `conv.u*` for `any to unsigned *` (where * is the target width)
+- Taking a few examples:
+  - `sbyte->nint` and `sbyte->nuint` use `conv.i` while `byte->nint` and `byte->nuint` use `conv.u` because they are all **widening**.
+  - `nint->byte` and `nuint->byte` use `conv.u1` while `nint->sbyte` and `nuint->sbyte` use `conv.i1` as a _pre-existing semantic_. These are both **narrowing** and so there isn't a specific need to pick `signed` (`i1`) or `unsigned` (`u1`), its just a semantic of what's already done.
```

Comment:
This bit should be reworded based on what I put as a follow up: https://github.com/dotnet/csharplang/pull/6059#discussion_r858027728

The semantic is that the stack type is `int32` not `byte` or `sbyte` and so we have to convert to the target sign so when its widened to `int32` on the stack, the value is still correct